dipy/segment/tests/test_metric.py

```diff
@@ -78,19 +83,25 @@ def dist(self, v1, v2):
         s_zero_mean = s - s_mean
         s_rotated = np.dot(M_rotation, s_zero_mean.T).T + s_mean
 
-        opposite = norm(np.cross(rot_axis, s_zero_mean), axis=1) / norm(rot_axis)
-        distances = np.sqrt(2*opposite**2 * (1 - np.cos(60.*np.pi/180.))).astype(dtype)
+        opposite = norm(np.cross(rot_axis, s_zero_mean),
+                        axis=1) / norm(rot_axis)
+        distances = np.sqrt(2 * opposite**2 *
+                        (1 - np.cos(60. * np.pi / 180.))).astype(dtype)
         d = np.mean(distances)
         assert_almost_equal(metric.dist(s, s_rotated), d, 5)
 
-        for s1, s2 in itertools.product(*[streamlines]*2):  # All possible pairs
-            # Extract features since metric doesn't work directly on streamlines
+        for s1, s2 in itertools.product(
+                *[streamlines] * 2):  # All possible pairs
+            # Extract features since metric doesn't work directly on
+            # streamlines
             f1 = metric.feature.extract(s1)
             f2 = metric.feature.extract(s2)
 
             # Test method are_compatible
             same_nb_points = f1.shape[0] == f2.shape[0]
-            assert_equal(metric.are_compatible(f1.shape, f2.shape), same_nb_points)
+            assert_equal(
```

Comment:
This can fit on two lines instead of three, pay attention to the indention though.